src/App.js

```diff
@@ -16,53 +16,53 @@ function App() {
   const [isLoading, setIsLoading] = useState(true);
   const [requestError, setRequestError] = useState(false);
 
-
-//  random number between 1 and 154
+  //  random number between 1 and 154
   function randomNumber() {
     // min and max included
     return Math.floor(Math.random() * 154 + 1);
   }
 
-
-// make api request on mount
+  // make api request on mount
   useEffect(() => {
```

Comment:
This works but looks ripe for refactoring into a hook - like `useFetch(url)` or something. You should be able to do the hook such that it returns data, loading, error and so you should't need these any more

```javascript
  const [apiData, setApiData] = useState([]);
  const [isLoading, setIsLoading] = useState(true);
  const [requestError, setRequestError] = useState(false);
```

Reply:
Do I update this branch with the changes?